examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -63,35 +63,38 @@ class _BottomAppBarDemoState extends State<BottomAppBarDemo> {
 
   // FAB Position
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndDocked = _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndDocked =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Attached - End',
     label: 'floating action button is docked at the end of the bottom app bar',
     value: FloatingActionButtonLocation.endDocked,
   );
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterDocked = _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterDocked =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Attached - Center',
     label: 'floating action button is docked at the center of the bottom app bar',
     value: FloatingActionButtonLocation.centerDocked,
   );
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndFloat= _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndFloat =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Free - End',
     label: 'floating action button floats above the end of the bottom app bar',
     value: FloatingActionButtonLocation.endFloat,
   );
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterFloat = _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterFloat =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Free - Center',
     label: 'floating action button is floats above the center of the bottom app bar',
     value: FloatingActionButtonLocation.centerFloat,
   );
 
   static void _showSnackbar() {
-    const String text =
-      "When the Scaffold's floating action button location changes, "
-      'the floating action button animates to its new position.'
-      'The BottomAppBar adapts its shape appropriately.';
+    const String text = "When the Scaffold's floating action button location changes, "
+        'the floating action button animates to its new position.'
+        'The BottomAppBar adapts its shape appropriately.';
```

Comment:
not a huge fan of either the original or the formatted code here but the original is mildly better.